pkg/utils/eventrecorder/eventrecorder.go

```diff
@@ -83,27 +77,12 @@ func New(rawK8SClient, cachedK8SClient client.Client) (*EventRecorder, error) {
 
 }
 
-// BroadcastEvent will raise event on aws-node with given type, reason, & message
-func (e *EventRecorder) BroadcastEvent(eventType, reason, message string) {
-
-	// Get aws-node pod objects with label & field selectors
-	labelSelector := labels.SelectorFromSet(labels.Set{labelK8sapp: awsNode})
-	fieldSelector := fields.SelectorFromSet(fields.Set{specNodeName: MyNodeName})
-	listOptions := client.ListOptions{
-		LabelSelector: labelSelector,
-		FieldSelector: fieldSelector,
-	}
+// BroadcastPodEvent will raise event on aws-node with given type, reason, & message
+func (e *EventRecorder) BroadcastPodEvent(eventType, reason, message string) {
```

Comment:
nit: maybe we can rename as 'SendPodEvent' to be consistent.

Reply:
Good call, this sounds better to me